lib/inventory_refresh/inventory_collection/index/proxy.rb

```diff
@@ -224,7 +228,7 @@ def assert_index(manager_uuid, ref)
             assert_relation_keys({named_ref(ref).first => manager_uuid}, ref)
           end
         rescue => e
-          #_log.error("Error when asserting index: #{manager_uuid}, with ref: #{ref} of: #{inventory_collection}")
+          logger.error("Error when asserting index: #{manager_uuid}, with ref: #{ref} of: #{inventory_collection}")
```

Comment:
This class has `include InventoryRefresh::Logging` already so good to use the `logger` here